Dockerfile.debian

```diff
@@ -0,0 +1,52 @@
+# Neolink Docker image build scripts
+# Copyright (c) 2020 George Hilliard
+# SPDX-License-Identifier: AGPL-3.0-only
+
+FROM docker.io/rust:buster AS build
+MAINTAINER thirtythreeforty@gmail.com
+
+ENV DEBIAN_FRONTEND=noninteractive
+RUN set -eux; \
+    apt-get update; \
+    apt-get install -y --no-install-recommends \
+    build-essential \
+    musl-dev gcc \
+    libgstrtspserver-1.0-dev \
+    libgstreamer1.0-dev \
+    libgtk2.0-dev \
+    libglib2.0-dev; \
+    apt-get clean -y; \
+    rm -rf /var/lib/apt/lists/* /var/cache/apt/*
+
+# Use static linking to work around https://github.com/rust-lang/rust/pull/58575
+ENV RUSTFLAGS='-C target-feature=-crt-static'
```

Comment:
Isn't this workaround only required on musl?

Reply:
Maybe because of this it didn't work without `musl-dev`. No idea.